gigascience/gigadb-website
# Update address in contact page

# Pull request for issue: #1993

This is a pull request for the following functionalities:

* Update address and phone number in contact page

## How to test?

Review http://gigadb.gigasciencejournal.com/site/contact
verify that image should also be updated to reflect new address
confirm that updated image is correct

## How have functionalities been implemented?

- updated visible address and phone number as indicated
- updated image and image alt to reflect new address

## Any issues with implementation?

--

## Any changes to automated tests?

--
